### PR TITLE
Fix dockerfile git tag outdated issue

### PR DESCRIPTION
**Pull request by MittWillson**
```
$ git describe --tags
```

will returns `v1.0.0` wich is wrong tag, the latest is `v1.0.7`

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,8 +3,8 @@ RUN apt-get update
 RUN apt-get install -y git && apt-get install -y curl
 RUN git clone https://github.com/Polkadex-Substrate/Polkadex
```
Comment (simonsso) [inline Dockerfile:4]: This will check out main branch...

File: Dockerfile [hunk 2]
```diff
@@ -3,8 +3,8 @@ RUN apt-get update
...
 RUN cd Polkadex && \
-    git checkout $(git describe --tags --abbrev=0) && \ # Gets the latest tag
-    curl -O -L https://github.com/Polkadex-Substrate/Polkadex/releases/download/$(git describe --tags --abbrev=0)/customSpecRaw.json && \ # Gets the latest Spec
+    git checkout $(git describe --tags $(git rev-list --tags --max-count=1)) && \
+    curl -O -L https://github.com/Polkadex-Substrate/Polkadex/releases/download/$(git describe --tags $(git rev-list --tags --max-count=1))/customSpecRaw.json && \
```
Comment (simonsso) [inline Dockerfile:7]: This is nolonger needed the configspec is stored in extras/custConfigSpec.json

**Comment by simonsso** [comment]
@MittWillson  How does this look ? https://github.com/Polkadex-Substrate/Polkadex/pull/396

**Comment by MittWillson** [comment]
@simonsso Looks great!

**Pull request closed without merging**